front/app/components/PostShowComponents/Comments/CommentSorting.tsx

```diff
@@ -25,7 +25,7 @@ export default class CommentSorting extends PureComponent<Props, State> {
   title = (<FormattedMessage {...messages.commentsSortTitle} />);
 
   sortOptions = [
-    { text: <FormattedMessage {...messages.oldestToNewest} />, value: '-new' },
+    { text: <FormattedMessage {...messages.oldestToNewest} />, value: 'new' },
```

Comment:
I think we got lost in translation here - the original intention was to show the oldest first as indicated by the messages name but later on it got translated to "most recent" which is the opposite and it makes more sense from a UX perspective, actually. So I'll rename the string to avoid confusion and keep the "most recent" functionality in place.